index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<head>
+  <link rel="stylesheet" href="./public/style.css">
+  <script src='./src/jQuery.js'></script>
+  <script src='./src/Game.js'></script>
+  <script src='./src/Frame.js'></script>
+  <script src='./src/interface.js'></script>
+</head>
+
+<h1>Bowling</h1>
+
+<button id=0>
+  0
+</button>
+<button id=1>
+  1
+</button>
+<button id=2>
+  2
+</button>
+<button id=3>
+  3
+</button>
+<button id=4>
+  4
+</button>
+<button id=5>
+  5
+</button>
+<button id=6>
+  6
+</button>
+<button id=7>
+  7
+</button>
+<button id=8>
+  8
+</button>
+<button id=9>
+  9
+</button>
+<button id=10>
+  10
+</button>
+
+<h2>Your Score: <span id='score'></span></h2>
+<h3 id='frames'></h3>
```

Comment:
Glad to see you have an up and running html page. 
I have not done the maths yet to check if the numbers are correct. 
I see that the numbers are only being shown at the end of each round. In this case it would be nice to get some feedback that when you are pressing buttons, something is actually happening (such as round number increasing or just a confirmation message) as it is not immediately apparent.